api/valor_api/backend/query/mapping.py

```diff
@@ -106,7 +100,7 @@ def _recursive_select_to_table_names(
         )
 
 
-def map_arguments_to_tables(args: tuple[Any]) -> set[TableTypeAlias]:
+def map_arguments_to_tables(*args) -> set[TableTypeAlias]:
```

Comment:
add back typing to args? args are still defined in the docstring (tuple[Any])

Reply:
*args is python magic and defines a variable number of inputs that will be formatted into a ordered tuple.

Ill update the docstring to say the following:

```
    Parameters
    ----------
    *args : tuple[Any]
        A variable length list of arguments from a selection statement.
```

Reply:
If you define the function with `def fn(args: tuple[Any])` the input has to be passed as a preformatted tuple.